domain/build.gradle

```diff
@@ -42,8 +42,10 @@ dependencies {
     compile (project(':repository')) {
         exclude group: 'org.mapstruct', module: 'mapstruct'
     }
-    compile "org.apache.commons:commons-lang3"
-    compile group: 'commons-validator', name: 'commons-validator', version: '1.6'
+    compile "org.apache.commons:commons-lang3:3.11"
+    compile (group: 'commons-validator', name: 'commons-validator', version: '1.6') {
+        exclude group: 'commons-beanutils', module: 'commons-beanutils'
```

Comment:
can we add global exclude instead of doing in individual file?

Reply:
We still need one of the version of these package, global exclude would not work.